go.mod

```diff
@@ -37,3 +36,7 @@ require (
 	golang.org/x/sys v0.0.0-20190310054646-10058d7d4faa // indirect
 	gopkg.in/yaml.v2 v2.2.2
 )
+
+// https://github.com/0robustus1/cobra/tree/350-add_fish_completion_support
+// https://github.com/spf13/cobra/pull/754
+replace github.com/spf13/cobra => ../cobra
```

Comment:
Note that this replacement will only work on your local machine as it's using a relative path.

Note also that replace statements are ignored by `go get` (see https://github.com/golang/go/issues/31173 and #289).

Reply:
@twpayne oh yeah sorry, my comment should've been a bit clearer that this is temporary while I wait for the upstream PR to be merged. Its also why I raised this a draft PR.

Reply:
Cool, yup got that this is a draft. Thanks for opening it so that people know that the work is in progress.

Reply:
I've updated `go.mod` to be useable outside of my machine 🙂